rootfs-builder/alpine/config.sh

```diff
@@ -24,6 +24,7 @@ INIT_PROCESS=kata-agent
 ARCH_EXCLUDE_LIST=()
 
 [ "$SECCOMP" = "yes" ] && PACKAGES+=" libseccomp"
+[ "$AGENT_TRACE" = "yes" ] && PACKAGES+=" socat"
```

Comment:
@jodh-intel Does this need to go to the Dockerfiles as well? I am not clear if the Dockerfiles are used when USE_DOCKER is true.

Reply:
Not really as the docker files are mostly static - we want `socat` to be added conditionally to avoid bloat at this stage.